### ISSUE TITLE
React Native warning Animated: 'useNativeDriver'

### ISSUE THREAD

**Issue by Girum** [shahen94/react-native-switch#93]
Hi. I used the library recently, and jad no problem. But, when I used today, the React NAtive throws the warning for useNativeDriver. Do you resolve this in a new PR, or has a way in the props to do that?

**Comment by f4z3k4s** [comment]
Experiencing the same.

**Comment by EvangelosVorgias** [comment]
Same. Warning states:
Animated: 'useNativeDriver' was not specified. This is a required option and must be explicitly set to 'true' or 'false'.

**Comment by sargsyanPicsart** [comment]
Experiencing the same.

**Comment by IgarMikhailov** [comment]
Experiencing the same.

**Comment by tlgync** [comment]
node_modules -> react-native-switch -> lib -> Switch.js 

Add useNativeDrive:false to these lines.

![Ekran Resmi 2021-08-07 14 46 28](https://user-images.githubusercontent.com/5376360/128599078-a169f8ba-050c-48af-8121-3e637f27ffd4.png)

**Comment by nkemcels** [comment]
@tlgync Hmm... Editing files in the node modules folder is not a good idea.

**Comment by tlgync** [comment]
> @tlgync Hmm... Editing files in the node modules folder is not a good idea.

you need to fork it.

**Comment by hupptechnologies** [comment]
> @tlgync Hmm... Editing files in the node modules folder is not a good idea.

@nkemcels Yes but using the patch-package we can have the permanent fix by changing code in node modules.

**Comment by petros-g** [comment]
library is not leveraging UI thread. **Use this to solve your issue:** [react-native switch](https://www.npmjs.com/package/@petros-g/react-native-switch)
